com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/steps/PublishConfigurationEntriesStepTest.java

```diff
@@ -81,7 +82,8 @@ public void setUp() throws Exception {
     public void prepareDao() throws Exception {
         for (ConfigurationEntry entry : exisitingConfigurationEntries) {
             Mockito.when(configurationEntryDaoMock.find(Mockito.matches(entry.getProviderNid()), Mockito.matches(entry.getProviderId()),
-                Mockito.matches(entry.getProviderVersion().toString()), Mockito.matches(entry.getTargetSpace()), Mockito.any(),
+                Mockito.matches(entry.getProviderVersion().toString()),
+                Mockito.any(CloudTarget.class), Mockito.any(),

```

Comment:
In this way the test no longer validates correct filtration based on the target. Can this be enhanced to take into consideration the ConfigurationEntry's target?